java/developer-guide/presentation-slide/convert-slide/_index.md

```diff
@@ -3,144 +3,255 @@ title: Convert Slide
 } finally {
     if (pres != null) pres.dispose();
 }
 ```
-## **Convert Slide to Image with Custom Size**
 
-Sometimes you need to get an image of a slide of a certain size. 
-The following example demonstrates this capability using one of the 
-[getThumbnail](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#getThumbnail-java.awt.Dimension-) method overloads:
+This sample code shows you how to convert the first slide of a presentation to a bitmap object using the [RenderToGraphics](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#renderToGraphics-com.aspose.slides.IRenderingOptions-java.awt.Graphics2D-) method:
 
 ``` java 
 Presentation pres = new Presentation("Presentation.pptx");
 try {
-    // Convert the first slide of the presentation to a Bitmap with the specified size
+	// Gets the presentation slide size
+	Dimension2D slideSize = pres.getSlideSize().getSize();
+
+	// Creates a BufferedImage with the slide size
+	BufferedImage slideImage = new BufferedImage((int) slideSize.getWidth(), (int) slideSize.getHeight(), BufferedImage.TYPE_INT_ARGB);
+	java.awt.Graphics graphics = slideImage.createGraphics();
+
+	// Renders the first slide to the Graphics object
+	try {
+		pres.getSlides().get_Item(0).renderToGraphics(new RenderingOptions(), (Graphics2D) graphics);
+	} finally {
+		if (graphics != null) graphics.dispose();
+	}
+
+	ImageIO.write(slideImage, "png", new File("Slide_0.png"));
+} finally {
+	if (pres != null) pres.dispose();
+}
+```
+
+{{% alert title="Tip" color="primary" %}} 
+
+You can convert a slide to a BufferedImage object and then use the object directly somewhere. Or you can convert a slide to a BufferedImage and then save the image in JPEG or any other format you prefer. 
+
+{{% /alert %}}  
+
+## **Converting Slides to Images with Custom Sizes**
+
+You may need to get an image of a certain size. Using an overload from the [GetThumbnail](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#getThumbnail-com.aspose.slides.IRenderingOptions-) or [RenderToGraphics](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#renderToGraphics-com.aspose.slides.IRenderingOptions-java.awt.Graphics2D-) method, you can convert a slide to an image with specific dimensions (length and width). 
+
+This sample code demonstrates the proposed conversion using the [GetThumbnail](https://apireference.aspose.com/slides/java/com.aspose.slides/ISlide#getThumbnail-java.awt.Dimension-) method in Java:
+
+``` java 
+Presentation pres = new Presentation("Presentation.pptx");
+try {
+    // Converts the first slide in the presentation to a Bitmap with the specified size
     BufferedImage bmp = pres.getSlides().get_Item(0).getThumbnail(new Dimension(1820, 1040));
-    {
-        // Save the image in JPEG format
-        ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
-    }
+	
+	// Saves the image in the JPEG format
+	ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
```

Comment:
Inconsistency in output image format. Please choose one from JPEG or PNG.

Reply:
Done